index.html

```diff
@@ -174,5 +175,37 @@ <h1>I'm Shimels, Glad to see you!</h1>
         </li>
       </ul>
     </section>
+    <section id="contact-form">
+      <h1>Contact me</h1>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it!
+      </p>
+      <form action="https://formspree.io/f/xyyqzzjg" method="post">
+        <input
+          type="text"
+          id="name"
+          name="user_name"
+          placeholder="Write your name here"
+          maxlength="30"
+          required
+        />
```

Comment:
👍 Functionally, your form works as requested in the project requirements! However, there are a few design-related adjustments that need attention:

- [ ] The border radius of each input element is currently set at 10px, whereas the design specifies a border radius of 8px. Please update your code to match the design and set the border radius to 8px.

- [ ] Additionally, each input field should have a 1px border with a specified color, as indicated in the design. Consider adding the border with the correct color to achieve a closer match with the design specifications.
![image](https://github.com/shimels1/portfolio_microverse/assets/55840999/b93be64b-3d6c-4634-ac4b-3ed0650d1ba3)

- [ ] As mentioned earlier, avoid using fixed width and height. Instead, adjust the padding to achieve the desired height of the input fields, in accordance with the design. Proper padding will ensure the input fields look aesthetically pleasing and in line with the design.

By making these design adjustments, your form will appear more polished and aligned with the project's visual requirements. Keep up the good work, and I'm confident your attention to detail will result in a beautifully designed form! 🎨📝🚀